TODO.txt

```diff
@@ -6,18 +6,15 @@ Version 0.17
 * Finalize ``skimage.future.graph`` API.
 * Finalize ``skimage.future.manual_segmentation`` API.
 
-Version 1.0
------------
-
+Version 0.20
+------------
 * consider removing the argument `coordinates` in
```

Comment:
There is also 

https://github.com/scikit-image/scikit-image/blob/f4978b11499e499cc006c417b3bf0ccf245ca72c/skimage/measure/_regionprops.py#L1295-L1309

Looking at the git blame and checking the SHA with `git tag --contains`, it's already deprecated since 0.16. :smile: Should I finally remove?!

Reply:
I would say yes! :wink:

Reply:
Done in d752adf.